containers/run_jenkins_controller.sh

```diff
@@ -38,15 +38,22 @@ if ! [ -d "${JCCACHE}" ] ; then
   # Make sure we have absolute path
   JCCACHE="$(cd "${JCCACHE}" || exit 1 ; pwd)"
 
-  if ! ( cd "${JCCACHE}" || exit 1
-    git clone -n --depth=1 --filter=tree:0 \
-      https://github.com/tiiuae/ci-public home
-    cd home || exit 1
-    git sparse-checkout set --no-cone jenkins
-    git checkout
-  ) ; then
-    echo "Failed to setup Jenkins config clone" >&2
-    exit 1
+  if test -n "${JCC_REPO}" && test -n "${JCC_BRANCH}" ; then
```

Comment:
Please use brackets instead of test, like elsewhere in this script, and like it's suggested in
bash script style guide.

Reply:
Ups, I were scripting stuff that expected the opposite, and my brain failed to switch back to ghaf style.